src/ralph/back_office/models.py

```diff
@@ -41,7 +47,92 @@ class Meta:
 
     @property
     def country_code(self):
-        return 'PL'
+        if self.owner:
+            iso2 = Country.name_from_id(int(self.owner.country)).upper()
+            return iso2_to_iso3(iso2)
+        return 'POL'
```

Comment:
But country field is a required field, so there's always some country will be assigned.
Default country is: pl